.github/dependabot.yml

```diff
@@ -5,10 +5,10 @@
 
 version: 2
 updates:
-  - package-ecosystem: "npm" # See documentation for possible values
-    directory: "/" # Location of package manifests
+  - package-ecosystem: 'npm' # See documentation for possible values
```

Comment:
Could we please add `guthub-actions` as well? This will be something like this:

```yaml
  - directory: '/'
    package-ecosystem: 'github-actions'
    schedule:
      interval: 'daily'
```

Reply:
Resolved in 63333ffc61a2.